src/era/secrets_template.py

```diff
@@ -0,0 +1,52 @@
+"""
+secrets_template.py: Example secrets file. Copy to 'secrets.py' and populate with actual values.
+DO NOT ADD new secrets.py to version control.
+"""
+
+import logging
+import socket
+from pathlib import Path
+
+import numpy as np
+
+#: AGOL Account Settings
+AGOL_ORG = 'https://utah.maps.arcgis.com'
+AGOL_USER = ''
+AGOL_PASSWORD = ''
```

Comment:
**question**: are we able to register this script as an app in agol and use client id and secret for authentication tokens rather than a real account?

Reply:
I've not done that before, I'll have to look into it. It makes sense.